plugin/src/main/java/app/cash/paraphrase/plugin/GenerateFormattedResources.kt

```diff
@@ -109,6 +110,8 @@ internal abstract class GenerateFormattedResources @Inject constructor() : Defau
     if (mergedResources.isNotEmpty()) {
       writeResources(namespace.get(), mergedResources)
         .writeTo(outputDirectory.get().asFile)
+    } else {
+      outputDirectory.asFileTree.files.forEach(File::delete)
```

Comment:
Can/should it just delete the whole directory instead?

Reply:
Thought about it, but the documentation on `File::delete` says that directories have to be empty before you can delete them:
> Deletes the file or directory denoted by this abstract pathname. If this pathname denotes a directory, then the directory must be empty in order to be deleted.